car-service/src/main/java/xws/team16/carservice/service/CarService.java

```diff
@@ -131,4 +129,139 @@ public MyImage extractImage(String string) throws SQLException {
         myImage.setType(type);
         return myImage;
     }
+
+    public Set<Car> getAllCarsByOwnersId(Long ownersID){
+        log.info("Car service - get all cars by owner's id (id: " + ownersID + ")");
+
+        User owner = userService.getUserById(ownersID);
+        //return carRepository.findAllByOwner(owner).orElseThrow(() -> new NotFoundException("Not one car was found."));
+        return carRepository.getAllByOwner(owner);
+        // ako ne nađe nijedna kola, da li vraća null ili prazan Set<Car> ???
```

Comment:
I think it returns empty set, you can check that in IntelliJ, if you assign return value to a variable and check if it's null, IntelliJ will prompt you that if is redundant and that variable is never null. So if you want it to return null check if set is empty, and return null if it is.